### ISSUE TITLE
Error creating cache

### ISSUE THREAD

**Issue by tsulatsitamim** [eugeneware/ffmpeg-static#64]
I try to install on docker node node:lts-slim

but gives an error:
```
> ffmpeg-static@4.2.7 install /app/node_modules/ffmpeg-static
> node install.js

Error creating cache: ENOENT: no such file or directory, mkdir '/root/.cache/ffmpeg-static-nodejs'
```

how to properly install it on docker? thanks

**Comment by maxill1** [comment]
same problem here:

`Error creating cache: ENOENT: no such file or directory, mkdir '/root/.cache/ffmpeg-static-nodejs'`

using docker buildx on github workflow ([this file](https://github.com/maxill1/JukeReSTainer/blob/master/.github/workflows/build-dev.yml)).

Running on my machine is fine

**Comment by derhuerst** [comment]
This fails because the install script expects `~/.cache` (`/root/.cache` in your case) to be present already.

**Comment by derhuerst** [comment]
I have fixed the underlying issue in version `8.2.1` of `@derhuerst/http-basic` (which is a dependency of `ffmpeg-static`). It should work now, and reopen if it doesn't.
